addons/sys_radio/stringtable.xml

```diff
@@ -161,21 +161,33 @@
         </Key>
         <Key ID="STR_ACRE_sys_radio_DefaultRadio_DisplayName">
             <English>Radio Item Replacement</English>
+            <German>Austausch des Funkgeräts</German>
+            <Italian>Sostituzione della radio</Italian>
         </Key>
         <Key ID="STR_ACRE_sys_radio_DefaultRadio_Description">
             <English>Default radio inventory items will be replaced with this type of radio.</English>
+            <German>Standard Funkinventargegenstände werden durch diesen Funkgerätetyp ersetzt.</German>
+            <Italian>Gli elementi dell'inventario radio predefiniti verranno sostituiti con questo tipo di radio.</Italian>
```

Comment:
You could also use `radio predefinite`, but I believe this is more descriptive.
```suggestion
            <Italian>Le radio vanilla nell'inventario verranno sostituite con questo tipo di radio.</Italian>
```

Reply:
Yours sounds better!